core/domain/user_validators.py

```diff
@@ -1,1606 +0,0 @@
-# coding: utf-8
-#
-# Copyright 2020 The Oppia Authors. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS-IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""Validators for user models."""
-
-from __future__ import absolute_import  # pylint: disable=import-only-modules
-from __future__ import unicode_literals  # pylint: disable=import-only-modules
-
-import datetime
-
-from core.domain import base_model_validators
-from core.domain import exp_domain
-from core.domain import learner_progress_services
-from core.domain import rights_manager
-from core.domain import user_domain
-from core.domain import user_services
-from core.domain import wipeout_service
-from core.platform import models
-import feconf
-import utils
-
-(
-    base_models, collection_models, email_models,
-    exp_models, feedback_models, question_models,
-    skill_models, story_models, suggestion_models,
-    topic_models, user_models
-) = models.Registry.import_models([
-    models.NAMES.base_model, models.NAMES.collection, models.NAMES.email,
-    models.NAMES.exploration, models.NAMES.feedback, models.NAMES.question,
-    models.NAMES.skill, models.NAMES.story, models.NAMES.suggestion,
-    models.NAMES.topic, models.NAMES.user
-])
-
-
-class UserSettingsModelValidator(base_model_validators.BaseUserModelValidator):
```

Comment:
What is the replacement for these validators? /cc @vojtechjelinek 

(Or do these only run asynchronously for now -- I assume new changes to entities are still being validated in the domain layer?)

Reply:
We're slowly building them up as beam job valiators, the full progress is being tracked on a [spreadsheet](https://docs.google.com/spreadsheets/d/1Pi5KzVE17iWhYU2NJXLr3n7HC1Dxt8JmQzH3ZvsxJFo/edit#gid=0).